sdk/core/core/inc/az_http_policy.h

```diff
@@ -27,6 +27,17 @@
 
 typedef struct az_http_policy az_http_policy;
 
+/**
+ * @brief options for retry policy
+ * max_retry = maximun number of retry intents before returning error
+ * delay = waiting time before retrying in miliseconds
+ *
+ */
+typedef struct {
+  size_t max_retry;
+  size_t delay;
```

Comment:
delay should not be `size_t`. Use fixed size types, like `uint16_t`. `size_t` is used ONLY to measure memory blocks and it depends on platform.

Reply:
updated